src/graphql/operations/proposals.ts

```diff
@@ -66,13 +66,15 @@ export default async function (parent, args) {
     params.push(verifiedSpaces);
   }
 
+  // TODO: remove part `p.id IN (?)` when flagged proposals are moved from laser DB to snapshot-sequencer DB
   if (where.flagged === true && flaggedProposals.length > 0) {
```

Comment:
Beware of the case where this condition will not be triggered if `flaggedProposals.length` is 0.

Refactoring to remove `p.id IN (?)` has to be done before disconnecting laser database

Reply:
I know, thank you 🙏